### ISSUE TITLE
[Feature] Rotate consoles/machines to face the player who constructed it

### ISSUE THREAD

**Issue by Fr-Dae** [unitystation/unitystation#10110]
### There is no other issue reporting the same problem.
- [X] I have searched the existing issues

when constructed, it is oriented in the direction of the player who placed/created it.
thus resolves the problem of orientation and rotation of frames, computers and machines.

**Comment by corp-0** [comment]
we just took a look at this issue with Daxam and the way it is done currently makes sense. The computer is facing the same way the computer frame it was built from is facing.

Maybe you can clarify is there is a problem with that?

**Comment by Fr-Dae** [comment]
I'm only talking about in-game, not via the unity editor.
When you build a computer, it will always be in the same direction, which sometimes isn't practical or pretty.

**Comment by Fr-Dae** [comment]
there is no option to rotate a frame

**Comment by corp-0** [comment]
Then that's the real issue

**Comment by Daxam12** [comment]
I will try to resolve this when I'm able to.

**Comment by Fr-Dae** [comment]
> Then that's the real issue

no shit? i often have fun making issues just to say hello to you?
of course it's all time , not very detailed, but the goal is always to improve US13 or bring SS13/tgmc functionality.

I take the beta testing of all the games I contribute to very seriously, and finding bugs or missing features is what makes it fun.

**Comment by corp-0** [comment]
No, you dum dum. But if you read the description, you are asking for something different. At no point in your description it says "you can not rotate computer frames", so you never actually described the real issue.

**Comment by Fr-Dae** [comment]
when constructed,(ingame)

 it is oriented in the direction of the player who placed/created it. (player, not dev in unity editor ) 

- thuis resolves the problem of orientation and rotation of frames, computers and machines.

some of your devs don't even bother to read the description and it's up to me to find an exact title that perfectly describes the situation... so I try to sum it all up in the title,

the placement of the frame is relative to the players
(like the stairs on minecraft)

and yes I'm dumdum but that's not the point

**Comment by corp-0** [comment]
you are terrible at writing descriptions and understanding the replies. It is not our fault.

> thuis resolves the problem of orientation and rotation of frames, computers and machines.

Sure, that is an approach, but it is not the one we think we should take. The computer inherits the facing from the frame it was built from. The **real issue** is that frames don't have a way to be rotated by players, so that is what we have to add.

**Comment by Fr-Dae** [comment]
> It is not our fault.

in fact if everyone makes an effort 
me to better explain what I mean in feature or bug requests,
and you (the whole team), to read and understand what I mean (even with a lame title) we'll , it's a win-win


and finding titles that sum it all up is super difficult

**Comment by corp-0** [comment]
you are still not following.

Issue: computer frames can't be rotated
Your solution: make the built computer spawn facing the player that built it.

This does work, but it not a direct solution to the issue. If you have a place that is too dark because it is missing a light, you don't make the sun brighter to compensate, you add the missing light.

In this case, the frame is missing a feature which is being able to be rotatable, so the solution is to add that missing feature.

**Comment by Fr-Dae** [comment]
i agree
